hack/tools/catalogs/list-compatible-bundles

```diff
@@ -90,7 +90,7 @@ function filter-by-regex-if-necessary() {
         end'
 }
 
-cat - | select-bundle-documents | that-support-allnamespace-install-mode | that-dont-have-dependencies | group-versions-by-package-name | filter-by-regex-if-necessary
+cat - | select-bundle-documents | that-support-allnamespace-install-mode | that-dont-have-dependencies | extract-olm-package-property | group-versions-by-package-name | filter-by-regex-if-necessary
 
 echo "NOTE: OLM v1 currently only supports bundles that support AllNamespaces install model, don't have dependencies, and don't contain webhooks" >&2
 echo "WARNING: These results may include bundles with webhooks, which are incompatible" >&2
```

Comment:
@perdasilva NIT ^ We need to give a space at the end of the file here.

Reply:
Fixed!